src/server.h

```diff
@@ -1576,6 +1574,9 @@ struct redisServer {
         long long samples[STATS_METRIC_SAMPLES];
         int idx;
     } inst_metric[STATS_METRIC_COUNT];
+    long long stat_reply_buffer_shrinks; /* Total number of buffer output buffer shrinks */
+    long long stat_reply_buffer_expends; /* Total number of output buffer expends */
```

Comment:
Are these reset in `resetServerStats()`? If not please add them.

Reply:
@yoav-steinberg you are totally correct. I missed that one. will be fixed ASAP